### PR TITLE
docs(RHTAPWATCH-1261): prepare o11y docs for handoff

### PR DESCRIPTION
**Pull request by yftacherzog**
This changes mainly includes minor updates for things changed since the documentation was written. Additionally, some of the content was removed as it's being moved to the SOP repository.

### CODE REVIEW

File: components/monitoring/README.md [hunk 1]
```diff
@@ -1,214 +1,233 @@
...
-      │   └── kustomization.yaml
-      ```
-
-  2. Create a folder in your team's repository to maintain the dashboard configuration (e.g. grafana)
+New dashboards can be added through the user interface, preconfigured in the
+infra-deployments repository, or imported from other projects.
+
+### Create a Dashboard
+
+1. [Create a dashboard](https://grafana.com/docs/grafana/v9.0/dashboards/)
```
Comment (avi-biton) [inline components/monitoring/README.md:68]: I see that our Grafana version is already 10.4.3

```suggestion
1. [Create a dashboard](https://grafana.com/docs/grafana/v10.4/dashboards/)
```

File: components/monitoring/prometheus/README.md [hunk 2]
```diff
@@ -90,16 +86,16 @@ data:
       retentionSize: 1GiB
 ```
 
-#### Observability Operator (OBO) Prometheus
-Federates the Platform and UWM Prometheus instances.
+### Observability Operator (OBO) Prometheus
+This is instance federates the Platform and UWM Prometheus instances.
```
Comment (avi-biton) [inline components/monitoring/prometheus/README.md:90]:
```suggestion
This instance federates the Platform and UWM Prometheus instances.
```

File: components/monitoring/README.md (hunk 1 above)
Comment (yftacherzog) [reply components/monitoring/README.md]: done

File: components/monitoring/prometheus/README.md (hunk 2 above)
Comment (yftacherzog) [reply components/monitoring/prometheus/README.md]: done

**Review by avi-biton — APPROVED** [review]
lgtm

**Pull request merged**